ui/src/main/js/view/widgets/dialog/style.less

```diff
@@ -1,13 +1,16 @@
 .cbwf-dialog {
   border-radius: @cbwf-border-radius;
   border: solid 1px @dark-border-color;
+  border: solid 1px var(--panel-border-color);
   background-color: @light-body-background-color;
+  background-color: var(--background);
```

Comment:
Is this variable only found on the dark-theme? Shouldn't we only use variables defined on Jenkins core?

Reply:
https://github.com/jenkinsci/jenkins/blob/master/war/src/main/less/abstracts/colors.less#L21

Reply:
👍

Reply:
I know I'm a bit late, but 
```css
background-color: @light-body-background-color;
background-color: var(--background);
```
Looks like a bug to me.

I also can't see where var(--background) is defined

Reply:
@VirtualTim no worries mate. Maybe provide a PR fixing it?

Reply:
I don't really know enough about how this works for a PR. My guess is that the intention was to do `background-color: var(--background, @light-body-background-color);`, but I'm just guessing.